ding/entry/serial_entry_offline.py

```diff
@@ -0,0 +1,86 @@
+from typing import Union, Optional, List, Any, Tuple
```

Comment:
add test for this PR:
- unittest (add `test_cql` in `ding/entry/test_serial_entry.py`, which ensures the unittest and coverage(`ding/entry/serial_entry_offline.py`, `ding/policy/cql.py`) )
- algotest (add `test_cql` in `ding/entry/test_serial_entry_algo.py`, which ensures the convergence of cql algorithm (discrete in cartpole, continuous in pendulum) )

Reply:
current cql algorithm is only implemented in continuous environments.

Reply:
then just continuous for this PR